site/about/use-cases/e-23.qmd

```diff
@@ -0,0 +1,164 @@
+---
+title: "E-23"
+date: last-modified
+---
+
+OSFI Guideline E-23 (Enterprise-Wide Model Risk Management for Deposit-Taking Institutions) contains the most comprehensive treatment of AI among major MRM regulations, reflecting the 2027 effective date and the anticipated evolution of AI. The guideline focuses on achieving specific outcomes rather than following prescribed processes.
+
+**Effective date:** May 1, 2027
+
+## Overview
+
+E-23 establishes expectations for federally regulated deposit-taking institutions in Canada to manage model risk effectively. It covers traditional MRM requirements while incorporating forward-looking considerations for AI and machine learning models.
+
+## Traditional MRM requirements
+
+E-23 aligns with established MRM frameworks, requiring:
+
+- **Model inventory** — Comprehensive registry of all models
+- **Development standards** — Clear documentation of model design and implementation
+- **Validation framework** — Independent validation and effective challenge
+- **Ongoing monitoring** — Performance tracking and periodic review
+- **Governance** — Board oversight and clear accountability
+
+::: {.callout title="Related documentation"}
```

Comment:
Mainly a comment for @validbeck: I'll turn these links into margin footnotes to follow our standard ...